vueapp/src/components/MemberUpsert.vue

```diff
@@ -1,38 +1,133 @@
 <template>
-    <v-card
-      prepend-icon="mdi-account-circle"
-      text="Blah blah blah"
-      :title=member.name
-      class="mx-auto"
-      width="90%"
-    >
-      <v-card-actions>
-        <v-btn 
-        text="Initiate challenge"
-        @click="initiateChallenge"
-        ></v-btn>
-        <v-spacer></v-spacer>
+  <v-card
+    v-if="currentScreen == Screen.INITIALSCREEN"
+    prepend-icon="mdi-account-circle"
+    text="Blah blah blah"
+    :title="challenger.name"
+    class="mx-auto"
+    width="90%"
+  >
+    <v-card-actions>
+      <v-btn text="Initiate challenge" @click="goToChallengeSetUp"></v-btn>
+      <v-spacer></v-spacer>
+      <v-btn text="DONE" @click="done"></v-btn>
+    </v-card-actions>
+  </v-card>
+  <v-card
+    v-if="currentScreen == Screen.SETUPSCREEN"
+    prepend-icon="mdi-account-circle"
+    text="Blah blah blah"
+    :title="challenger.name"
+    class="mx-auto"
+    width="90%"
+  >
+    <div>
+      <v-select
+        label="Target level"
+        :items="levelStore.allLevels"
+        item-title="name"
+        v-model="targetLevel"
+      >
+      </v-select>
+    </div>
+    <v-card-actions>
+      <v-btn
```

Comment:
Could save the user a step/click by including the challengers selection component on the same display as the level select.  Trigger on the level selection and update the membes available in the challenger selection.